ksqldb-serde/src/test/java/io/confluent/ksql/serde/connect/ConnectDataTranslatorTest.java

```diff
@@ -319,6 +320,19 @@ public void shouldTranslateMissingStructFieldToNull() {
     assertThat(row.get("STRUCT"), is(nullValue()));
   }
 
+  @Test
+  public void shouldReturnTimeType() {
+    // Given:
+    final ConnectDataTranslator connectToKsqlTranslator = new ConnectDataTranslator(Time.SCHEMA);
+
+    // When:
+    final Object row = connectToKsqlTranslator.toKsqlRow(Time.SCHEMA, new Date(100L));
```

Comment:
This is a `Date`. Why is the result a `Time`?

Reply:
The Connect time types (Time, Date, Timestamp) store data as `java.util.Date`. `ConnectDataTranslator ` converts data from Connect to KSQL.